examples/game.ink

```diff
@@ -1,3 +1,5 @@
+-> back_in_london
```

Comment:
Nice improvement, thank you,
would it work in this example, if we just removed both the `-> back_in_london` and  `=== back_in_london ===` and the .ink file would start with `We arrived into London...`?

Reply:
Yep, that works as well. I’ve only seen this being done [in the tutorial](https://www.inklestudios.com/ink/web-tutorial/80-days-example.txt) but I suppose you can do this for content that you don’t have to divert to in the future.

Reply:
ok, thanks, 
anyway if you use this in a game or anything I'd be happy to hear about it :)